dora/core/common/src/main/java/alluxio/grpc/GrpcUtils.java

```diff
@@ -554,6 +567,7 @@ public static alluxio.grpc.MountPointInfo toProto(MountPointInfo info) {
    */
   public static alluxio.grpc.WorkerInfo toProto(WorkerInfo workerInfo) {
     return alluxio.grpc.WorkerInfo.newBuilder().setId(workerInfo.getId())
+        .setIdentity(workerInfo.getIdentity().toProto())
         .setAddress(toProto(workerInfo.getAddress()))
```

Comment:
so im assuming this is used only for blockmaster calls to get worker list from master right? we don't need to support protobuf ser/de (toproto / from proto ) at all IMO.

Reply:
> don't need to support protobuf ser/de (toproto / from proto ) at all IMO.

Currently a lot of tests depend on protobuf ser/de. Unless we can ignore all these tests, I don't think this can be skipped.